src/services/api/peers.js

```diff
@@ -93,11 +94,24 @@ app.factory('Peers', ($timeout, $cookies, $location, $q, $rootScope) => {
      * @private
      * @memberOf Peer
      * @method check
+     * @notifyUser {bool} - Should an error toast be displayed on error?
      */
-    check() {
+    check(notifyUser) {
```

Comment:
I think it makes perfect sense to notify the user of any error which happens during connections. so this parameter seems to be redundant.